src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -233,10 +233,14 @@ public boolean verifyTitle(String title) {
     return true;
   }
 
-  public void verifyURLcontains(String givenString) {
+  public void verifyURLcontains(String exptectedString) {
     String currentURL = driver.getCurrentUrl();
-    Assertion.assertStringContains(currentURL.toLowerCase(), givenString.toLowerCase());
-    PageObjectLogging.log("verifyURLcontains", "current url is the same as expetced url", true);
+    if (Assertion.assertStringContains(currentURL.toLowerCase(), exptectedString.toLowerCase())) {
+      PageObjectLogging.log("verifyURLcontains", "current url contains " + exptectedString, true);
+    } else {
+      PageObjectLogging
+          .log("verifyURLcontains", "current url doesn't contain " + exptectedString, true);
```

Comment:
It is, indeed, `true` that in this case `"current url doesn't contain " + exptectedString`, but it will be printed in log on a green background. Therefore, as a reader I would be confused if it's OK or not. Is it?
You could remove this else, and create a separate method `verifyURLDoesNotContain()`, which suggests what you are expecting?